openml-utils/src/test/java/com/feedzai/openml/util/provider/AbstractProviderModelBaseTest.java

```diff
@@ -25,6 +25,7 @@
 import com.feedzai.openml.provider.model.MachineLearningModelLoader;
 import com.feedzai.openml.util.algorithm.MLAlgorithmEnum;
 import com.google.common.collect.ImmutableList;
+import com.google.common.primitives.Doubles;
```

Comment:
This import is unused, right? Please clean it up

Reply:
This is used in the method AbstractProviderModelBaseTest#classifyIndexOfMaxScoresValue() to obtain the index of maximum value from a list of double, e.g. _Doubles.asList(scores).indexOf(maxScore)_.

But I just reviewed the code, and there is a method that uses the ArrayUtils. For this reason I will remove this import and update the method mentioned above to use the following _Arrays.asList(ArrayUtils.toObject(scores)).indexOf(maxScore)_.